### DISCUSSION TITLE
Allow searching for users

### DISCUSSION THREAD

**Discussion by minas1** [trakt/trakt-api#747] [Q&A]
It would be really useful to have an API to search for users.

It should be able to search for:

* slugs
* display names

This would be really useful in allowing people to find their friends without going to trakt.tv website.

**Comment by kevincador** [comment]
+1

**Comment by kawa89** [comment]
Any update on this?
There is a feature on the Web, so it would be good to have it in the API :)

**Comment by javikr** [comment]
Any update??
